tests/e2e/vmgroupbasic_test.go

```diff
@@ -0,0 +1,240 @@
+
+	// Create a volume from _DEFAULT vmgroup's vm - operation should succeed
+	out, err := dockercli.CreateVolume(s.config.DockerHosts[0], s.volumeNames[0])
+	c.Assert(err, IsNil, Commentf(out))
+
+	// Check if volume was successfully created
+	isAvailable := verification.CheckVolumeAvailability(s.config.DockerHosts[0], s.volumeNames[0])
+	c.Assert(isAvailable, Equals, true, Commentf("Volume %s is not available after creation", s.volumeNames[0]))
+
+	// Create a volume from non-default vmgroup's vm - operation fails
+	out, err = dockercli.CreateVolume(s.config.DockerHosts[1], s.volumeNames[1]+"@"+s.config.Datastores[1])
+	c.Assert(err, Not(IsNil), Commentf(out))
+
+	// Check volume was not created
+	isAvailable = verification.CheckVolumeAvailability(s.config.DockerHosts[1], s.volumeNames[1]+"@"+s.config.Datastores[1])
+	c.Assert(isAvailable, Equals, false, Commentf("Unexpected behavior: Volume %s is successfully created "+
+		" even though vmgroup [%s] does not have access rights for that datastore", s.volumeNames[1], vmGroupName))
+
+	// Set the create privilege on the vmgroup  for specified datastore
+	out, _ = admincli.AddCreateAccessForVMgroup(s.config.EsxHost, vmGroupName, s.config.Datastores[1])
+	isDatastoreAccessible = admincli.IsDSAccessibleForVMgroup(s.config.EsxHost, vmGroupName, s.config.Datastores[1])
+	c.Assert(isDatastoreAccessible, Equals, true, Commentf("Datstore %s is not accessible for vmgroup %s .", s.config.Datastores[1], vmGroupName))
+
+	// Create a volume from non-default vmgroup's vm - operation should succeed this time
+	// as access-rights have been changed to True
+	out, err = dockercli.CreateVolume(s.config.DockerHosts[1], s.volumeNames[1]+"@"+s.config.Datastores[1])
+	c.Assert(err, IsNil, Commentf(out))
+
+	// Check if volume was successfully created
+	isAvailable = verification.CheckVolumeAvailability(s.config.DockerHosts[1], s.volumeNames[1]+"@"+s.config.Datastores[1])
+	c.Assert(isAvailable, Equals, true, Commentf("Volume %s is not available after creation", s.volumeNames[1]))
+
+	// verify able to delete  volume
+	out, err = dockercli.DeleteVolume(s.config.DockerHosts[1], s.volumeNames[1]+"@"+s.config.Datastores[1])
+	c.Assert(err, IsNil, Commentf(out))
+
+	// Remove the create privilege on the non-default vmgroup for specified datastore
+	admincli.RemoveCreateAccessForVMgroup(s.config.EsxHost, vmGroupName, s.config.Datastores[1])
+
+	// Create a volume from non-default vmgroup's vm - operation should fail
+	volumeName := inputparams.GetUniqueVolumeName(vmGroupName)
+	out, err = dockercli.CreateVolume(s.config.DockerHostNames[1], volumeName+"@"+s.config.Datastores[1])
+	c.Assert(err, Not(IsNil), Commentf(out))
+
+	out, err = dockercli.DeleteVolume(s.config.DockerHosts[0], s.volumeNames[0])
+	c.Assert(err, IsNil, Commentf(out))
+
+	admincli.RemoveVMFromVMgroup(s.config.EsxHost, vmGroupName, s.config.DockerHostNames[1])
```

Comment:
@govint  I am already deleting a vmgroup and verifying that its successfully deleted in test : ```TestOrphanedVolumesVmgroups```
Do I still need to do it again? It will make the test even longer